controller/src/main/java/com/softserve/borda/controllers/BoardController.java

```diff
@@ -454,21 +454,19 @@ public Board getBoard(@PathVariable Long id) {
     }
 
     @PostMapping("/users/invitations/{receiverUsername}/boards/{boardId}/roles/{userBoardRoleId}")
-
     public ResponseEntity<Object> createInvitation(Authentication authentication,
                                                           @PathVariable String receiverUsername,
                                                           @PathVariable Long boardId,
                                                           @PathVariable Long userBoardRoleId) {
-        var user = userService.getUserByUsername(authentication.getName());
+        User sender = userService.getUserByUsername(authentication.getName());
```

Comment:
Do not make changes that are not related to a task.
it can be a cause of merge conflicts.

Reply:
Yes, my bad. There was some error after merge and after fixing it I also refactored it a bit. Will try not to do it in the future.